libnopegl/src/hud.c

```diff
@@ -531,13 +531,13 @@ static void draw_block_graph(struct hud *s,
 {
     const int64_t graph_h = graph_max - graph_min;
     const float vscale = (float)rect->h / (float)graph_h;
-    const int start = (d->pos - d->count + d->nb_values) % d->nb_values;
+    const size_t start = (d->pos - d->count + d->nb_values) % d->nb_values;
```

Comment:
`count` is always larger or equal to `pos`, so the subtraction goes into unsigned underflow. The arithmetic still works out but do we want this or should we for example reorder the operation to put the subtract at the end to avoid the underflow?

Reply:
Let's give `pos=3`, `count=10`, and `nb_values=10`.

In unsigned space:
- `(pos-count+nb_values)%nb_values = (3-10+10)%10 = 3`
- `(pos-count)%nb_values = (3-10)%10 = 18446744073709551609%10 = 9`

So nah.